src/connectors/hierarchical-menu/__tests__/__snapshots__/connectHierarchicalMenu-test.js.snap

```diff
@@ -0,0 +1,93 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`connectHierarchicalMenu routing getWidgetSearchParameters should add the refinements according to the UI state provided 1`] = `
+SearchParameters {
+  "advancedSyntax": undefined,
+  "allowTyposOnNumericTokens": undefined,
+  "analytics": undefined,
+  "analyticsTags": undefined,
+  "aroundLatLng": undefined,
+  "aroundLatLngViaIP": undefined,
+  "aroundPrecision": undefined,
+  "aroundRadius": undefined,
+  "attributesToHighlight": undefined,
+  "attributesToRetrieve": undefined,
+  "attributesToSnippet": undefined,
+  "disableExactOnAttributes": undefined,
+  "disjunctiveFacets": Array [],
+  "disjunctiveFacetsRefinements": Object {},
+  "distinct": undefined,
+  "enableExactOnSingleWordQuery": undefined,
+  "facets": Array [],
+  "facetsExcludes": Object {},
+  "facetsRefinements": Object {},
+  "getRankingInfo": undefined,
+  "hierarchicalFacets": Array [
+    Object {
+      "attributes": Array [
+        "category",
+        "subCategory",
+      ],
+      "name": "category",
+      "rootPath": null,
+      "separator": " > ",
+      "showParentLevel": true,
+    },
+  ],
+  "hierarchicalFacetsRefinements": Object {
+    "category": Array [
+      "path",
+    ],
+  },
+  "highlightPostTag": undefined,
+  "highlightPreTag": undefined,
+  "hitsPerPage": undefined,
+  "ignorePlurals": undefined,
+  "index": "",
+  "insideBoundingBox": undefined,
+  "insidePolygon": undefined,
+  "length": undefined,
+  "maxValuesPerFacet": 10,
+  "minProximity": undefined,
+  "minWordSizefor1Typo": undefined,
+  "minWordSizefor2Typos": undefined,
+  "minimumAroundRadius": undefined,
+  "numericFilters": undefined,
+  "numericRefinements": Object {},
+  "offset": undefined,
+  "optionalFacetFilters": undefined,
+  "optionalTagFilters": undefined,
+  "optionalWords": undefined,
+  "page": 0,
+  "query": "",
+  "queryType": undefined,
+  "removeWordsIfNoResults": undefined,
+  "replaceSynonymsInHighlight": undefined,
+  "restrictSearchableAttributes": undefined,
+  "snippetEllipsisText": undefined,
+  "synonyms": undefined,
+  "tagFilters": undefined,
+  "tagRefinements": Array [],
+  "typoTolerance": undefined,
+}
+`;
+
+exports[`connectHierarchicalMenu routing getWidgetState should add an entry equal to the refinement 1`] = `
+Object {
+  "hierarchicalMenu": Object {
+    "category": Array [
+      "path",
+    ],
+  },
+}
+`;
+
+exports[`connectHierarchicalMenu routing getWidgetState should not overide other entries in the same namespace 1`] = `
+Object {
+  "hierarchicalMenu": Object {
+    "category": Array [
```

Comment:
Shouldn't it conains  otherCategory: ['path'], too?

Reply:
Yes

Reply:
This is fixed.

Reply:
Fixed :)